config/routes.rb

```diff
@@ -46,7 +46,7 @@
     resources :change_requests, only: :create
   end
   resources :addresses do
-    resources :change_requests, only: :create
+    resources :change_requests, only: %i[create destroy]
```

Comment:
It doesn't seem like we need to expose an actual `destroy` route, since to delete a resource, we issue the creation of a change request that itself represents the requested deletion.